google/cloud/pubsub_v1/subscriber/_protocol/dispatcher.py

```diff
@@ -303,6 +316,10 @@ def drop(
         Args:
             items: The items to drop.
         """
+        for item in items:
+            if item.opentelemetry_data:
+                item.opentelemetry_data.set_subscribe_span_result("dropped")
```

Comment:
I think we call this "expired" in the design, but if you prefer dropped to match the client library method, that's fine.

Reply:
Sure, renamed to "expired"